app/presenters/base.presenter.js

```diff
@@ -534,6 +535,20 @@ function leftPadZeroes(number, length) {
   return number.toString().padStart(length, '0')
 }
 
+/**
+ *
+ * @param {number} paginationTotal - the total count from the fetch requests 'total'
+ * @param {number} currentAmount - the current amount of the items (the last page would show 'Showing 7 of 32')
+ * @param {string} message - the message to add to the end of the string (Showing x of y 'returns')
+ */
+function paginationShowingXofY(paginationTotal, currentAmount, message) {
```

Comment:
We could add this to the pagination presenter ? 

As an additional variable on the exported object ? 

'showingMessage' 

this would require passing in the 'message'